<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUserWidget</class>
 <widget class="QWidget" name="NewUserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Login">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>40</y>
     <width>311</width>
     <height>175</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
border:none;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QLabel" name="label_top">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:  url(:/icons-lib/screen/screensection_top.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_topLeft">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/icons-lib/screen/screensection_topleft.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_left">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:  url(:/icons-lib/screen/screensection_left.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_bottomLeft">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/icons-lib/screen/screensection_bottomleft.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_Bottom">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:  url(:/icons-lib/screen/screensection_bottom.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_topRight">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/icons-lib/screen/screensection_topright.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_right_3">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:  url(:/icons-lib/screen/screensection_right.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_bottomRight">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/icons-lib/screen/screensection_bottomright.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_Login_1">
      <property name="styleSheet">
       <string notr="true">QGroupBox{border:none;
background-image:  url(:/icons-lib/screen/screensection_middle.png);
}
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_userId">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>3</y>
         <width>80</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>User ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Password">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>43</y>
         <width>80</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_lineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>2</y>
         <width>198</width>
         <height>37</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox
{
border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_right_2">
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="logo.qrc">:/icons-lib/screen/textfield_left.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_LoginID">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
    border:none;
	background-image:  url(:/icons-lib/screen/textfield_middle.png);

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_right">
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="logo.qrc">:/icons-lib/screen/textfield_right.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_lineEdit_1">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>42</y>
         <width>198</width>
         <height>37</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox
{
border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_right_7">
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="logo.qrc">:/icons-lib/screen/textfield_left.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Password">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
    border:none;
	background-image:  url(:/icons-lib/screen/textfield_middle.png);

}</string>
          </property>
          <property name="maxLength">
           <number>30</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_right_8">
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="logo.qrc">:/icons-lib/screen/textfield_right.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_fingerPrint">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>81</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string>Finger Print</string>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>70</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_save">
       <property name="geometry">
        <rect>
         <x>186</x>
         <y>120</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Save [F2]</string>
       </property>
       <property name="icon">
        <iconset resource="logo.qrc">
         <normaloff>:/icons-lib/screen/ok.png</normaloff>:/icons-lib/screen/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="geometry">
        <rect>
         <x>86</x>
         <y>120</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel [F4]</string>
       </property>
       <property name="icon">
        <iconset resource="logo.qrc">
         <normaloff>:/icons-lib/screen/cancel.png</normaloff>:/icons-lib/screen/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_Password_2">
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>84</y>
         <width>60</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>security</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>146</x>
         <y>90</y>
         <width>140</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Password only</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fingerprint only</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Both</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>220</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons-lib/screen/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="title_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{border:none; border-image: url(:/icons-lib/screen/topbar_long.png);}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>NEW USER </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>7</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons-lib/screen/login.png);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/topbar/images/topbar/patients_icon.png</pixmap>
   </property>
  </widget>
  <zorder>label_Background</zorder>
  <zorder>groupBox_Login</zorder>
  <zorder>title_groupBox</zorder>
  <zorder>label_2</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_LoginID</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_fingerPrint</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
